### ISSUE TITLE
Как установить...

### ISSUE THREAD

**Issue by qua-rus** [RestorerZ/RuVim#2]
Здравствуйте.
Взялся осваивать редактор.
Если возможно - черкните как установить справку RuVim в системе (Linux).
Расположение директорий для файлов нашёл, но толку от этого мало. В нэте тоже не нашёл ничего об этом...
Заранее спасибо.

**Comment by RestorerZ** [comment]
@qua-rus ,
Просто скопируйте содержимое архива туда, где находятся рабочие файлы редактора Vim.
В UNIX-подобных системах это, как правило, домашний каталог пользователя.
Для того, чтобы при запросе открывалась справка на русском языке, установите параметр «helplang»:
 `:set helplang =ru,en`
Установить его можно или прямо в командной строке редактора, или в файле «.vimrc».
Чтобы открыть какой‐нибудь файл справки можно воспользоваться следующей командой
 `:help usr_01@ru`
, набрав её в командной строке редактора.
Вкратце, как‐то так. Спрашивайте, если что‐то будет непонятно.
Удачного Viminga!

**Comment by qua-rus** [comment]
Здравствуйте.
Я в целом разобрался в вопросе и по ходу перевел часть файлов справки, необходимых для более полноценного функционирования русской справки и изучения Vim. 
Файлы - во вложении, предлагаю включить их в документацию RuVim (перевод в основном машинный - м.б. уточните).
Краткое руководство по внедрению файлов русской справки в общую справку Vim я изложил в файле **russian.rux**, в разделе "Локализация".
[help+.zip](https://github.com/RestorerZ/RuVim/files/13693536/help%2B.zip)

**Comment by qua-rus** [comment]
Исправил несколько неточностей в файлах... архив во вложении.
[help+.zip](https://github.com/RestorerZ/RuVim/files/13694990/help%2B.zip)

**Comment by qua-rus** [comment]
Исправил несколько неточностей в файлах... архив во вложении.
[help+.zip](https://github.com/RestorerZ/RuVim/files/13694990/help%2B.zip)

**Comment by qua-rus** [comment]
Исправил несколько неточностей в файлах... архив во вложении.
[RuVim+.zip](https://github.com/RestorerZ/RuVim/files/13696105/RuVim%2B.zip)

**Comment by qua-rus** [comment]
Исправил несколько неточностей в файлах... архив во вложении.
[RuVim+.zip](https://github.com/RestorerZ/RuVim/files/13696183/RuVim%2B.zip)

**Comment by qua-rus** [comment]
Исправил несколько неточностей в файлах... архив во вложении.
[RuVim+.zip](https://github.com/RestorerZ/RuVim/files/13696215/RuVim%2B.zip)

**Comment by qua-rus** [comment]
Исправил несколько неточностей в файлах... архив во вложении.
[RuVim+.zip](https://github.com/RestorerZ/RuVim/files/13696260/RuVim%2B.zip)

**Comment by RestorerZ** [comment]
@qua-rus , хорошо, что разобрались.
За сделанные Вами переводы спасибо! Отличная работа! Я посмотрю всё более внимательно, и позже включу в пакет «RuVim».

**Comment by qua-rus** [comment]
Здравствуйте.
Переводы чем-то не понравились?.. так и не увидел их в пакете.
Я правил файлы (не помню что) и добавил еще 1: 
[RuVim+.zip](https://github.com/RestorerZ/RuVim/files/15486788/RuVim%2B.zip)

**Comment by RestorerZ** [comment]
@qua-rus , нет, но, к сожалению, сейчас навалились некоторые срочные дела. Рассчитываю, что удастся решить их в ближайшее время и я вернусь к этому проекту.
Благодарю за проделанную Вами работу.
